pkg/generator/fluentd/output/kafka/sasl.go

```diff
@@ -0,0 +1,22 @@
+package kafka
+
+type SaslOverSSL bool
+
+func (s SaslOverSSL) Name() string {
+	return "kafkaSaslOverSSLTemplate"
+}
+
+func (s SaslOverSSL) Template() string {
+	enabled := `{{define "kafkaSaslOverSSLTemplate" -}}
```

Comment:
Or to put it a bit shorter

return fmt.Sprintln(){{`define "kafkaSaslOverSSLTemplate" -}}
sasl_over_ssl %t
{{- end}}`, s)

Reply:
done.